jobs/meta_stats.rb

```diff
@@ -6,11 +6,11 @@
 SCHEDULER.every '1h', :first_in => '1s' do |job|
 	backend = GithubBackend.new()
 	repos = backend.get_repos(
-		:orgas=>(ENV['ORGAS'].split(',') if ENV['ORGAS']), 
-		:repos=>(ENV['REPOS'].split(',') if ENV['REPOS']),
-		:since=>ENV['SINCE'],
+		orgs: (ENV['ORGS'].split(',') if ENV['ORGS']),
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.